New footer UI with Info pages loaded dynamically

smitpatwa opened the pull request:
![screenshot from 2016-07-08 14-33-11](https://cloud.githubusercontent.com/assets/10924779/16682618/0b48b25e-4519-11e6-8ffb-1dcdc52cb824.png)

saavrabh commented on gnowsys-ndf/gnowsys_ndf/ndf/static/ndf/scss/_app_styles.scss:
```diff
@@ -460,50 +460,96 @@ body{
 
 
```
Colors used in footer should be accordingly  particular theme colors.Right now you have used Clix style colors.

saavrabh commented on gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer.html:
```diff
@@ -1,52 +1,75 @@
+    <div class="footer-section container content-section">
+        <div class="row" style="padding-left:0px;">
+            <div class="footer_content">
+                <img class="footer-logo" src="/static/ndf/css/themes/clix/logo.svg">
```
Please use `{{site.logo}}` in order to get logo as per settings,Don't use static logo path.

smitpatwa commented:
1. I have added the logo src in the footer to be {{site.LOGO}} 
2. To change the backgroung according to theme, I have set the background color in app_styles.scss to be same as background in header in app_styles.scss. And added the clix background for footer in clix_styles.scss. Please let me know if anything else was to be done for this.

saavrabh commented:
Good,site logo and footer is fixed.One more suggestion is when you hover on any of links in footer.Color is dark blue and text is not visible.Can you change the color so that is would be easy to read even after hover? apart from this all looks fine for me.

likecs commented:
Repository, Groups and Partners webpage updated and all their styling is made common. Headers issue fixed. Cards issue while zooming-in and out also fixed. Hover issues reported in footer also fixed.

saavrabh commented on gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer1.html:
```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load get_latest_git_hash from simple_filters %}
```
Can you explain why this footer1.html template is added? We are already using footer.html for footer

smitpatwa replied:
footer1.html contains the old footer html. It can be ignored now.

kedar2a commented:
This commit has been merged in [alpha](https://github.com/gnowledge/gstudio/tree/alpha) branch.
